playground/frontend/playground_components/assets/translations/en.yaml

```
@@ -49,6 +49,13 @@ widgets:
   
   closeButton:
     label: 'Close'
+  
+  feedback:
+    bad: 'Bad Experience'
+    good: 'Good Experience'
```

```suggestion
    negative: 'Bad Experience'
    positive: 'Good Experience'
```
to match the enum values.